ciphey/basemods/Decoders/brainfuck.py

```diff
@@ -0,0 +1,172 @@
+from typing import Optional, Dict, List, Tuple
```

Comment:
Can you run the whole file through Python Black? It's a good formatting style that we try to use. I can do it for you if you want.

Reply:
No problem. Running it through now produced some long lines, so as long as you're okay without perfect PEP8 then that should be fine.